config/schema/indexes/mainstream.json

```diff
@@ -25,7 +25,6 @@
     "service_manual_topic",
     "service_standard_report",
     "tax_tribunal_decision",
-    "utaac_decision",
-    "vehicle_recalls_and_faults_alert"
```

Comment:
I think this means you can delete the type definition in vehicle_recalls_and_faults_alert.json, as well as some fields in field_definitions.json.

Reply:
Updated, thanks. That's a lot of code for one document type!